tests/test_radosbench.py

```diff
@@ -0,0 +1,61 @@
+import logging
+import random
+import sys
+log = logging.getLogger(__name__)
+
+
+def run(**kw):
+    ''' A task for radosbench
+
+    '''
+    log.info("Running radosbench task")
+    ceph_pods = kw.get('ceph_pods')  # list of pod objects of ceph cluster
+    config = kw.get('config')
+
+    clients = []
+    role = config.get('role', 'client')
+
+    for cpod in ceph_pods:
+        if role in cpod.roles:
+            clients.append(cpod)
+
+    idx = config.get('idx', 0)
+    client = clients[idx]
+    pg_num = config.get('pg_num', 64)
+    op = config.get('op', 'write')
+    cleanup = 'cleanup'
+    if config.get('cleanup'):
```

Comment:
cleanup = config.get('cleanup', 'cleanup')

Reply:
Sorry for the confusion. it supposed to be a bool. Changed it accordingly.